static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,219 @@
+staying on the external drive. DVC will still be able to track them properly.
+
+## Make the data directory accessible
+
+For this to work, first you have to make sure that you can read and write the
+data directory `/mnt/external-drive/`. The most straightforward way to do this
+is by setting proper ownership and permissions to it, like this:
+
+```dvc
+$ sudo chown <username>: -R /mnt/external-drive/
+$ chmod u+rw -R /mnt/external-drive/
+```
+
+## Start a DVC project and setup an external cache
+
+An _external_ <abbr>cache</abbr> is called so because it resides outside of the
+workspace directory. Let's create a directory for it on `/mnt/external-drive/`:
+
+```dvc
+$ mkdir -p /mnt/external-drive/dvc-cache
+```
+
+Now you can initialize a <abbr>project</abbr> on your home directory and
+configure it to use the external cache directory:
+
+```dvc
+$ cd ~/project/
+$ git init
+$ dvc init
+
+$ dvc config cache.dir /mnt/external-drive/dvc-cache
+$ rm -rf .dvc/cache/
+
+$ git add .dvc/config
+$ git commit -m 'Initialize DVC with external cache'
+```
+
+<details>
+
+### Transfer the content of the cache to the external directory
+
+In this example we are removing the default cache directory `.dvc/cache/`
+because we just initialized the project and we know that it is empty (there's
+nothing stored in it). If we had an existing project, we could preserve the
+content of the cache by moving it to the new directory:
+
+```dvc
+$ mv -a .dvc/cache/* /mnt/external-drive/dvc-cache/
+$ rm -rf .dvc/cache/
+```
+
+</details>
+
+If you check the config file you should see something like this:
+
+```dvc
+$ cat .dvc/config
+[cache]
+dir = /mnt/external-drive/dvc-cache
+```
+
+## Tracking external dependencies and outputs
+
+Now, when you refer to the data files and directories, you have to use their
+**absolute path**. The <abbr>DVC-files</abbr> will be created on the project
+directory, and you can track their modifications with `git` as usual.
+
+For example let's say that the raw data files are on `/mnt/external-drive/raw/`
+and you are cleaning them up. You could do it like this:
+
+```dvc
+$ dvc add /mnt/external-drive/raw
+
+$ dvc run -f clean.dvc \
+          -d /mnt/external-drive/raw \
+          -o /mnt/external-drive/clean \
+          ./cleanup.py /mnt/external-drive/raw /mnt/external-drive/clean
+```
+
+<details>
+
+### Using an environment variable for the data path
+
+In a real life situation probably you would declare an environment variable
+`DATA_PATH=/mnt/external-drive` and use it to shorten the command options, like
+this:
+
+```dvc
+$ dvc add $DATA_PATH/raw
+
+$ dvc run -f clean.dvc \
+          -d $DATA_PATH/raw \
+          -o $DATA_PATH/clean \
+          ./cleanup.py $DATA_PATH/raw $DATA_PATH/clean
+```
+
+</details>
+
+If you check the contents of the files `raw.dvc` and `clean.dvc` you will notice
+that their `path:` field refers to the external directories:
+
+```dvc
+$ cat raw.dvc
+```
```

Comment:
Please use "If you check the contents of `raw.dvc` (and `clean.dvc`) you will notice that the `path` field refers to the external directories:" and remove `dvc` block below (leave `yaml` only), removing also `clean.dvc` `dvc` and `yaml` blocks.

Reply:
Fixed in b3d5c6b9 and f345245f.